### ISSUE TITLE
Just curious, for line 528 and line 530 in models.py, why return x without calculation?

### ISSUE THREAD

**Issue by hildazzz** [p0p4k/vits2_pytorch#45]
```
x_ = torch.cat([x0, x1], 1)
logdet = torch.sum(logs, [1, 2])
return x, logdet
```

**Comment by p0p4k** [comment]
Yes, you are right, we can do that. However, see this [comment](https://github.com/jaywalnut310/vits/issues/46#issuecomment-1091389379) and this [comment](https://github.com/jaywalnut310/vits/issues/6#issuecomment-862751364) for reference. If we decide to use a non-volume preserving normalizing flow, we will need the `logdet` for NLL calculations. I think the authors might have had experimented with it and just left it in the code for re-usability.

**Comment by p0p4k** [comment]
Hi, I completely misread your question earlier and you are right, it is a typo. It should be `x` instead of `x_` in the first line. Thanks!!
